cl/api/templates/webhooks-docs-vlatest.html

```diff
@@ -293,6 +294,65 @@ <h3 id="search-alerts">Search Alert Events</h3>
+  </p>
+  <p>The webhook event is a JSON hash with two keys, <code>webhook</code> and <code>payload</code>. <code>payload</code> has three keys:</p>
+  <ul>
+    <li>
+      <p><code>disabled_alerts</code> (Automatically Disabled Alerts):</p>
+      <p>Contains a list of docket alerts that have been disabled automatically by our system on terminated cases that haven’t had updates for over 180 days. You can re-enable these alerts if they were disabled by mistake.</p>
+    </li>
+    <li>
+      <p><code>very_old_alerts</code> (Old Terminated Cases):</p>
+      <p>Contains a list of docket alerts terminated cases that were last updated about 180 days ago. These alerts will be disabled in a week if you don’t take action. If you are still monitoring these cases please disable and re-enable these alerts. Doing so will keep them running for another 180 days.</p>
+    </li>
+    <li>
+      <p><code>old_alerts</code> (Terminated about 90 Days Ago):</p>
+      <p>Contains a list of docket alerts on terminated cases that were last updated about 90 days ago. If you are no longer monitoring these cases, you may want to disable these alerts.</p>
+    </li>
+  </ul>
+  <p>The shape of the data is thus:</p>
+  <pre>{
+   "payload":{
+      "old_alerts":[
+         {
+            "id":1,
+            "date_created":"2022-09-23T19:53:36.903277-07:00",
+            "date_last_hit":null,
+            "secret_key":"ehT7V9rmnBNIOV6rTMmMH0x6EvxeA0nYXfpN3Ks3",
+            "alert_type":1,
+            "docket":1
+         }
+      ],
+      "very_old_alerts":[...],
+      "disabled_alerts":[...]
+   },
+   "webhook":{...}
+}</pre>
+  <p><code>old_alerts</code>, <code>very_old_alerts</code>, and <code>disabled_alerts</code> as mentioned above contain a list of docket alert objects based on the <a href="{% url 'rest_docs' %}#docket-alerts-endpoint">Docket Alerts API</a> and have all the same fields. </p>
+  <h4>Disabling and re-enabling docket alerts</h4>
+  <p>If you want to disable and re-enable a docket alert to keep them running for another 180 days or just re-enable a docket alert that was disabled by mistake, you can do it by sending a patch to the <a href="{% url 'rest_docs' %}#docket-alerts-endpoint">Docket Alerts</a> API endpoint.</p>
+  <p>Disabling a Docket Alert:</p>
+  <pre>curl -X PATCH \
+  --data 'alert_type=0' \
+  --header 'Authorization: Token {% if user.is_authenticated %}{{ user.auth_token }}{% else %}&lt;your-token-here&gt;{% endif %}' \
+  "{% get_full_host %}{% url "docket-alert-list" version="v3" %}{id}/"
+</pre>
+<p>Enabling a Docket Alert:</p>
+  <pre>curl -X PATCH \
+  --data 'alert_type=1' \
+  --header 'Authorization: Token {% if user.is_authenticated %}{{ user.auth_token }}{% else %}&lt;your-token-here&gt;{% endif %}' \
+  "{% get_full_host %}{% url "docket-alert-list" version="v3" %}{id}/"
+</pre>
```

Comment:
I'm not sure the PATCH approach below works for re-upping, but if not, I think that's where the focus should be in this doc:

```suggestion
  <p><code>old_alerts</code> and <code>disabled_alerts</code> contain a list of docket alert objects based on the <a href="{% url 'rest_docs' %}#docket-alerts-endpoint">Docket Alerts API</a> and have all the same fields. </p>
  <h4>Re-upping a docket alert</h4>
  <p>If a case has been dormant for a long time and you wish to continue monitoring it, you must re-up the alert. To do this, simply send an HTTP PATCH request to the <a href="{% url 'rest_docs' %}#docket-alerts-endpoint">Docket Alerts</a> API endpoint:</p>
  <pre>curl -X PATCH \
  --data 'alert_type=1' \
  --header 'Authorization: Token {% if user.is_authenticated %}{{ user.auth_token }}{% else %}&lt;your-token-here&gt;{% endif %}' \
  "{% get_full_host %}{% url "docket-alert-list" version="v3" %}{id}/"
</pre>
  <p>Doing the above will tell our system that the alert was recently modified and thus should not be disabled for another six months.
  </p>
  <p>A similar request can be used to disable any docket alert:</p>
  <pre>curl -X PATCH \
  --data 'alert_type=0' \
  --header 'Authorization: Token {% if user.is_authenticated %}{{ user.auth_token }}{% else %}&lt;your-token-here&gt;{% endif %}' \
  "{% get_full_host %}{% url "docket-alert-list" version="v3" %}{id}/"
</pre>
```

Reply:
Yes, I tested it, and your PATCH approach re-upping the docket alert without toggling the `alert_type`, works to update the `date_modified` so the alert got another 180 days.

Docs updated.